lisa/runner.py

```diff
@@ -105,6 +109,11 @@ def close(self) -> None:
         if self._log_handler:
             remove_handler(self._log_handler)
             self._log_handler.close()
+        self._log.debug(f"Runner finished in {self._timer.elapsed()}s.")
```

Comment:
This line should be above the close lines. So that this line can appear in runner's log file, if there is.